examples/ticks/centered_ticklabels.py

```diff
@@ -38,11 +38,12 @@
 ax.xaxis.set_major_formatter(ticker.NullFormatter())
 ax.xaxis.set_minor_formatter(dates.DateFormatter('%b'))
 
-for tick in ax.xaxis.get_minor_ticks():
-    tick.tick1line.set_markersize(0)
-    tick.tick2line.set_markersize(0)
-    tick.label1.set_horizontalalignment('center')
+# Remove the tick lines
+ax.tick_params(axis='x', which='minor', tick1On=False, tick2On=False)
 
+# Align the minor tick label
+for label in ax.get_xticklabels(minor=True):
+    label.set_horizontalalignment('center')
 imid = len(r) // 2
 ax.set_xlabel(str(r.date[imid].item().year))
-plt.show()
+fig.show()
```

Comment:
I think we basically always want `plt.show()` in examples. See the note in https://matplotlib.org/devdocs/api/figure_api.html#matplotlib.figure.Figure.show

Reply:
For me it resulted in two windows open. Not sure why, but the reason I changed it.

Reply:
At least that is what I imagined. Not anymore though...